### DISCUSSION TITLE
Rats

### DISCUSSION THREAD

**Discussion by pairofscissors** [cloewen8/Corntopia#6] [Ideas]
First off, rats.
Rats are little, fast, nuisances. They eat your corn and spread diseases among your crops. Eating corn can slightly or **massively** decrease corn production depending on their population. Rats can be combated with rodenticides (pesticides for rodents like rats) or with traps. Animals like cats can also stop rats. Oh and also, I doubt customers would want to buy crops from a farm that's know for having rats.

**Comment by cloewen8** [comment]
Separated each feature into its own discussion to make talking about each easier.

- Pesticides: https://github.com/cloewen8/Corntopia/discussions/8
- GMOs: https://github.com/cloewen8/Corntopia/discussions/9
